Resita/index.html

```diff
@@ -0,0 +1,213 @@
+    <link rel="icon" href="../Resita/favicon-16x16.png" type="image/x-icon" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+        integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <title>Resita</title>
+</head>
+
+<body>
+    <div id="top">
+        <ul>
+            <li id="emblem">
+                        <img src="../Resita/images/ROU_CS_Resita_CoA.png" alt="Resita Coat of Arms">
+            </li>
+            <li>
+                <a href="#history">
+                    <i class="fas fa-monument"></i> <span> History</span></a>
+                </li>
+            <li>
+                <a href="#location">
+                    <i class="fas fa-map-marker-alt"></i><span> Location</span></a>
+                </li>
+            <li>
+                <a href="#images">
+                    <i class="fas fa-image"></i><span> Images</span></a>
+                </li>
+            <li>
+                <a href="#sights">
+                    <i class="fas fa-camera"></i><span> Sights</span></a>
+                </li>
+            
+        </ul>
+    </div>
+    <div class="box1">
+        <img src="../Resita/images/resita2.jpg" alt="Resita at Night">
+        <h1>RESITA</h1>
+    </div>
+    <div class="h2">
+        <h2 id="history">History</h2>
+    </div>
+    <div class="box2">
+        <a href="../Resita/images/grebla-resita.jpg" target="_blank"><img src="../Resita/images/grebla-resita.jpg"
+                alt="Grebla Power Plant Resita"></a>
+        <div>
+            <p>Reșița (Romanian pronunciation: [ˈreʃit͡sa]; German: Reschitz;<span class="extratxt">Hungarian: Resicabánya; Croatian: Ričica;
+            Czech: Rešice; Serbian: Решица/Rešica; Turkish: Reşçe) is a city in western Romania and the capital of Caraș-Severin County, in the Banat region.Its 2011 population was 73,282.The name of Reșița might come from the Latin recitia, meaning "cold spring", as the historian Nicolae Iorga once suggested, presuming that the Romans gave this name to Resita,from a water spring on the Doman valley.</span></p>
+            <button class="button"><span><a href="https://en.wikipedia.org/wiki/Re%C8%99i%C8%9Ba" target="_blank">Read More</a></span></button>
+        </div>
+    </div>
+    <div class="h2">
+        <h2 id="h2"><span id="location">Location</span></h2>
+    </div>
+    <div class="box3">
+        <iframe
+            src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d35600.35490013474!2d21.86901507326111!3d45.30965171296632!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47502a14d6a718d9%3A0xb62b9e8261fdbec8!2sResita%2C+Romania!5e1!3m2!1sen!2sde!4v1556802287465!5m2!1sen!2sde"
+            width="800" height="600" frameborder="0" style="border:0" allowfullscreen></iframe>
+    </div>
+    <div class="h2">
+        <h2 id="images">Images</h2>
+    </div>
+    <div class="box4">
```

Comment:
What if we had 45 boxes.
Will you just keep on counting.
Try just calling them boxes or rows instead. Generic classes so you can add the same styling. And then if you wanna do separate styling just add an id named something.
In general avoid names like box45